data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,653 @@ mission "Heliarchs Buy Jump Drive Mission"
+				decline
+			label "literature collectors violent"
+			`	"Collectors of literature, we are. Curious about your race, and the books they have written, we are, so for you, a mission we have," says the lead Kimek, who then introduces themselves as Kachachee. "Literature of many kinds, we read, from science fiction to tales of alien worlds outside of our homeland.`
+				goto "literature collector choice"
+			label "literature collectors"
+			`	"Greetings, human friend. Collectors of literature, we are. Curious about your race, and the books they have written, we are, so for you, a mission we have," says a Kimek, who then introduces themselves as Kachachee. "Literature of many kinds, we read, from science fiction to tales of alien worlds outside of our homeland.`
+			label "literature collector choice"
+			`	"A small mission, it is. Knowledge from your race, we seek. Pay you <payment> and very grateful, we would be, if accept this mission, you will," they explain.`
+			choice
+				`	"I'm in. What do you want me to collect?"`
+				`	"Sorry, I'm not interested."`
+					decline
+			`	"Anything. A translation device, we have. To translate the human language into our own, we are able. Tales of vile oppressors and the heroic stands against them, always popular amongst our peoples, the topic is. Stories of brave battles, tyrants and rebels. Histories of humanity's resistance movements, we seek."`
+			label decision
+			choice
+				`	"Why do you want this knowledge? What is your group about?"`
+					goto "group question"
+					to display
+						not "coalition: asked about kachachee"
+				`	"I see. I will bring you all the knowledge you want."`
+				`	"I'm not interested. Sorry."`
+					decline
+			`	"Thank you. Enlighten us and the rest of the Coalition, your actions will. A safe trip, we hope you have." They give you instructions on where to find them.`
+			branch "don't have"
+				not "human cultural data"
+			`	You recall that your ship has a datachip containing a copy of Alexandria's archives, which may be what the group wants. You walk to your ship, make a copy of the datachip, and return to the group.`
+			`	Kachachee then approaches you as you hand them the datachip. "Very speedy, your return was."`
+			choice
+				`	"Happy to help."`
+				`	"I hope you enjoy whatever my race has to offer for you."`
+			action
+				event "coalition citizens discover fangvin" 14 27
+				payment 180000
+			`	"Thank you. Enlighten me and my friends, this datachip will. And as promised, a chip of <payment>, we have." Kachachee places it in your hand.`
+				decline
+			label "group question"
+			action
+				set "coalition: asked about kachachee"
+			`	"A group who likes to read books together, we are. Science fiction, I believe the term is, especially."`
+				goto decision
+			label "don't have"
+			`	On <stopovers> there is a library with complete copies of all human history. You return to your ship and prepare for the journey there to acquire one.`
```

Comment:
In other missions where you're asked for the Alexandria archives, you don't know where the archives are without being told.

Reply:
I think this is a bit more specific.
```suggestion
			`	A quick search of the Worlds Wide Web reveals that the library of <stopovers> has data chips containing a complete database of all human culture and history. You return to your ship and prepare for the journey there to acquire one.`
```